src/wp-admin/includes/image.php

```diff
@@ -660,6 +660,19 @@ function _wp_filter_image_sizes_additional_mime_type_support( $sizes, $attachmen
 		'post-thumbnail' => true,
```

Comment:
Let's remove `post-thumbnail` from this list in favor of https://github.com/WordPress/wordpress-develop/pull/3028/files#diff-4d28b6652f0dab208c19aa45e1cf09f3c967060d8cc77502bfcd473a3216b612R351 which you've already added.

Reply:
👍🏼 removed in 8853869e95d08c92868a5398cfc47623534cc5d4